www/css/main.diary.css

```diff
@@ -216,6 +216,22 @@ a.item-content {
   overflow: hidden;
   box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
   display: flex;
+  flex-wrap: wrap;
+}
+
+.diary-card.short {
```

Comment:
I was using this for place list items because they don't need to be as tall as trips.
I also though they should have a different color to differentiate from trips, and have the content centered like the Sydney team's wireframes showed

Reply:
Makes sense. Can you add comments to the css for the locations where the change was not obvious?
It will help the next person who needs to work on the code...